docs/content/guides/operator/formal-snapshot.mdx

```diff
@@ -0,0 +1,88 @@
+
+## Enabling formal snapshots
+
+Full nodes do not take formal snapshots by default. To enable this feature:
+
+1. Stop your node, if it's running.
+2. Open your fullnode.yaml config file.
+3. Add an entry to the config file for `state-snapshot-write-config`. Using Google's GCS service as an example:
+   ```yaml
+   state-snapshot-write-config:
+     object-store-config:
+       object-store: "GCS"
+       bucket: "<BUCKET-NAME>"
+       google-service-account: "<SERVICE-ACCOUNT-FILEPATH>"
+       object-store-connection-limit: 20
+    concurrency: 5
+   ```
+   - `object-store`: The remote object store to upload snapshots. Set as Google's `GCS` service in the example.
+   - `bucket`: The GCS bucket name to store the snapshots.
+   - `google-service-account`: Path to a GCS service account json file containing signing credentials.
+   - `aws-region`: Region where buck exists.
+   - `object-store-connection-limit`: Number of simultaneous connections to the object store.
+4. Currently, formal snapshots leverage database checkpoints (this will change soon). Add an entry to the config file for `db-checkpoint-config`. Using Amazon's S3 service as an example:
+   ```yaml
+   db-checkpoint-config:
+     perform-db-checkpoints-at-epoch-end: true
+     perform-index-db-checkpoints-at-epoch-end: true
+     object-store-config:
+       object-store: "S3"
+       bucket: "<BUCKET-NAME>"
+       aws-access-key-id: “<ACCESS-KEY>”
+       aws-secret-access-key: “<SHARED-KEY>”
+       aws-region: "<BUCKET-REGION>"
+       object-store-connection-limit: 20
+   ```
+   - `object-store`: The remote object store to upload snapshots. Set as Amazon's `S3` service in the example.
+   - `bucket`: The S3 bucket name to store the snapshots.
+   - `aws-access-key-id` and `aws-secret-access-key`: AWS authentication information with write access to the bucket.
+   - `aws-region`: Region where buck exists.
+   - `object-store-connection-limit`: Number of simultaneous connections to the object store.
+4. Save the sui-node.yaml file and restart the node.
+
+## Restoring from formal snapshots
+
+The full process of downloading, verifying, and restoring from a formal snapshot is handled by a single command in `sui-tool`. After running the snapshot restore tooling, the local node state reflects the canonical state at the end of the epoch to which you restored, at which point you can start the node normally. To restore from a snapshot:
+
+1. Download the `latest` version of `sui` for your operating system.
+1. Run `sui-tool download-db-snapshot --formal`, passing in the bucket name and authentication credentials as environment variables, and providing the bucket name, bucket type, epoch to restore to, and genesis blob.
```

Comment:
@ronny-mysten does the numbering here display correctly? I see that the numbered lists are repeating numbers